src/vivarium/framework/components/parser.py

```diff
@@ -20,11 +20,12 @@
 3. Importing and instantiating the actual components
 
 """
-from typing import Any, Dict, List, Tuple, Union
+from typing import Dict, List, Tuple, Union
 
 from vivarium.config_tree import ConfigTree
 from vivarium.framework.utilities import import_by_path
 
+from ... import Component
```

Comment:
I know you said pycharm is doing this, but is this really somethig we want to start doing?

Reply:
I'm not really sure what's happening here, but when I tried to type it as `from vivarium import Component`, I got a run-time error.